julia/mongoose/Project.toml

```diff
@@ -0,0 +1,2 @@
+[deps]
+Mongoose = "494b707d-6180-4d60-8018-fdbee1ff8085"
```

Comment:
is their a way to pin version without using uuid ? I mean it is hard to track when dealing with version updates